src/styles.ts

```diff
@@ -0,0 +1,74 @@
+import { css } from 'lit-element';
```

Comment:
we generally refer to the actual lit-element file in other files, so this should be changed to the same: lit-element/lit-element.js